bot/exts/evergreen/emojis.py

```diff
@@ -0,0 +1,39 @@
+import textwrap
+
+from discord import Color, Embed, Emoji
+from discord.ext import commands
+
+from bot.utils.time import time_since
+
+
+class Emojis(commands.Cog):
+    """Has commands related to emojis."""
+
+    @commands.group(name="emoji", invoke_without_command=True)
+    async def emojis_group(self, ctx: commands.Context, emoji: Emoji) -> None:
+        """A group of commands related to emojis."""
+        # No parameters = same as invoking info subcommand
```

Comment:
```suggestion
        # No parameters = same as invoking info subcommand
```
I don't think this comment adds anything, ctx.invoice(info_command) speaks for itself.

Reply:
As @thomaspet mentioned, this comment doesn't add anything, so it is not needed.
```suggestion
```